src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -140,37 +140,9 @@ enum class InConvFlowRegime
     Num
 };
 
-enum class SurfacesType
-{
-    Invalid = -1,
-    AllExteriorSurfaces,
-    AllExteriorWindows,
-    AllExteriorWalls,
-    AllExteriorRoofs,
-    AllExteriorFloors,
-    AllInteriorSurfaces,
-    AllInteriorWindows,
-    AllInteriorWalls,
-    AllInteriorRoofs,
-    AllInteriorCeilings,
-    AllInteriorFloors,
-    Num
-};
-
-constexpr std::array<std::string_view, static_cast<int>(SurfacesType::Num)> SurfacesTypeNamesUC{"ALLEXTERIORSURFACES",
-                                                                                                "ALLEXTERIORWINDOWS",
-                                                                                                "ALLEXTERIORWALLS",
-                                                                                                "ALLEXTERIORROOFS",
-                                                                                                "ALLEXTERIORFLOORS",
-                                                                                                "ALLINTERIORSURFACES",
-                                                                                                "ALLINTERIORWINDOWS",
-                                                                                                "ALLINTERIORWALLS",
-                                                                                                "ALLINTERIORROOFS",
-                                                                                                "ALLINTERIORCEILINGS",
-                                                                                                "ALLINTERIORFLOORS"};
-void InitInteriorConvectionCoeffs(EnergyPlusData &state,
-                                  const Array1D<Real64> &SurfaceTemperatures,    // Temperature of surfaces for evaluation of HcIn
-                                  ObjexxFCL::Optional_int_const ZoneToResimulate // if passed in, then only calculate surfaces that have this zone
+void InitIntConvCoeff(EnergyPlusData &state,
```

Comment:
Another thing I did. ConvectionCoefficients were variously referred to as `ConvectionCoefficient`, `ConvCoef`, and `ConvCoeff`. Everything should be `ConvCoeff`. Also, interior and exterior coefficients were variously referred to as `Interior`/`Exterior`, `Int`/`Ext`, `Inside`/`Outside`, and `In`/`Out`. As much as I love In-N-Out, I went with `Int`/`Ext`. This is less consistent and I will finish this cleanup in the next pass.